### PR TITLE
[IMP] adapt README.md for odoo.sh

### PR DESCRIPTION
**Pull request by sts-odoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,14 @@ $ ./odoo-bin --upgrade-path=/path/to/upgrade-util/src,/path/to/other/upgrade/scr
 ```
 
 ### The alternative one
-On platforms where you dont manage odoo yourself, i.e. [Odoo.sh](https://www.odoo.sh/), you can install this package via pip:
+On platforms where you dont manage odoo yourself, you can install this package via pip:
 ```shell-session
 $ python3 -m pip install git+https://github.com/odoo/upgrade-util@master
 ```
-
+On [Odoo.sh](https://www.odoo.sh/) it is recommended to add it to the `requirements.txt` of your repository:
```
Comment (amigrave) [inline README.md:18]: I think we should create a section for Odoo.sh so we can link directly on it from the Odoo.sh UI and we know we can put specifics later if needed.

```md
### On Odoo.sh
```

@KangOl you're ok with this ?

**Comment by KangOl** [comment]
@robodoo r+

**Comment by amigrave** [comment]
@KangOl huh !? @sts-odoo have not made the changes yet !

**Comment by KangOl** [comment]
New PR ¯\\_(ツ)_/¯

File: README.md (hunk 1 above)
Comment (erl-odoo) [reply README.md:18]: @sts-odoo what's the advantage of adding `odoo_upgrade` in front? I've added it without it, works fine
image

**Pull request closed without merging**